cardano-node/src/Cardano/Node/Protocol/Types.hs

```diff
@@ -40,34 +37,21 @@ instance FromJSON Protocol where
     withText "Protocol" $ \str -> case str of
 
       -- The new names
-      "MockBFT" -> pure (MockProtocol MockBFT)
-      "MockPBFT" -> pure (MockProtocol MockPBFT)
-      "MockPraos" -> pure (MockProtocol MockPraos)
       "Byron" -> pure ByronProtocol
       "Shelley" -> pure ShelleyProtocol
       "Cardano" -> pure CardanoProtocol
 
       -- The old names
-      "BFT" -> pure (MockProtocol MockBFT)
-    --"MockPBFT" -- same as new name
-      "Praos" -> pure (MockProtocol MockPraos)
       "RealPBFT" -> pure ByronProtocol
       "TPraos" -> pure ShelleyProtocol
```

Comment:
We should replace `RealPBFT` and `Praos` in all the configuration files with `Byron` and `Shelley`. (Better yet, remove all outdated configuration files).

Reply:
How would you then differentiate between e.q. Praos and Praos with some featured backported from Ouroboros Chronos? It would still be "Shelley" era, but with different version of Ouroboros consensus...

Reply:
The block type, e.g., `ShelleyBlock`, determines the protocol, e.g., `TPraos`. Switching to a new protocol like Ouroboros Chronos means hard forking to a new block type.